python/utest/pyitt_native_mock/__init__.py

```diff
@@ -0,0 +1,8 @@
+from sys import modules
+
+from .patch import patch
+from .pyitt_native_mock import PYITT_NATIVE_MODULE_NAME
```

Comment:
It would also be nice to rename all `pyitt` namings in utest layer to `ittapi`

Reply:
changed